tools/ClassGen/NodeGen.cpp

```diff
@@ -708,6 +708,9 @@ int main(int argc, char **argv) {
 
 #include "Backends/CPU/CPUSpecificNodes.h"
 #include "Backends/OpenCL/OpenCLSpecificNodes.h"
+  // Add here external backend specific node headers.
+  // Example:
+  // #include "<Name>/ClassGen/<Name>SpecificNodes.h"
```

Comment:
Seems like even if a backend is disabled we will still include their backend-specific Nodes/Instrs, right? I suppose that's already true, but might be nice to disable that somehow.